server/test/lib/system/system.getNetworkMode.test.js

```diff
@@ -4,12 +4,20 @@ const sinon = require('sinon');
 const { fake, assert } = sinon;
 
 const proxyquire = require('proxyquire').noCallThru();
-
+const fs = require('fs');
 const { PlatformNotCompatible } = require('../../../utils/coreErrors');
 const DockerodeMock = require('./DockerodeMock.test');
 
+let fsError = false;
+const fsMock = {
+  ...fs,
+  access: (file, options, callback) => {
```

Comment:
You'll need to mock this as a async function

You can use fake.resolves() to do this (search in the codebase for `fake.resolves` and you will see some examples :)

Reply:
Hard to learn, is Docker test use the same

https://github.com/GladysAssistant/Gladys/blob/master/server/test/lib/system/system.isDocker.test.js#L10

Reply:
No! 

One example: https://github.com/GladysAssistant/Gladys/blob/dc25cf7ed3ac36a2962e899e4c152ffe10c2c6ae/server/test/services/mqtt/lib/saveConfiguration.test.js#L23